tests/Utils/ComponentsConfigurationUtils.php

```diff
@@ -2,11 +2,19 @@
 
 namespace Keboola\Test\Utils;
 
+use Keboola\StorageApi\Components;
 use Keboola\StorageApi\Options\Components\Configuration;
 
 trait ComponentsConfigurationUtils
 {
-    public function createConfiguration($components, $componentId, $configurationId, $name)
+    /**
+     * @param Components $components
+     * @param string $componentId
+     * @param string|int $configurationId
+     * @param scalar $name
```

Comment:
Proč je tu `scalar $name`? Name je ve validaci string a nikde sa to nepřetypovává. Jinak je to třída enom v testoch klidně mohl použít přímo typehinty ne docbloky.

Reply:
jo, pravda, upraveno